.github/workflows/ci.yml

```diff
@@ -64,8 +65,9 @@ jobs:
     strategy:
       matrix:
         rust: [
-            stable,
-            nightly
+          1.51.0,
```

Comment:
Do the tests really need to run on the MSRV?

I think for development we can assume that people have latest Rust and the dev dependencies are not compiled when the crate is included as a dependency.

Reply:
You right, removed the 1.51 strategy for test job.